### ISSUE TITLE
KillauraLegit crash (MC 1.8)

### ISSUE THREAD

**Issue by obiobiminecraft** [Wurst-Imperium/Wurst-Client-Bugs#239]
#### Description of the Bug

When I try to turn on KillaurLegit in a MultyPlayer server, my minecraft crashes. Here's the crash report:

```
---------------- Wurst Client Crash Report ----------------
---- Please report this crash at: wurst-client.tk/bugs ----
// At least you got a nice looking crash report!

Time: 2016.10.08. 15:00
Description: Firing Wurst event

java.lang.NullPointerException: Firing Wurst event
    at net.minecraft.entity.Entity.getDistanceToEntity(Entity.java:1351)
    at tk.wurst_client.mods.KillauraLegitMod.onUpdate(KillauraLegitMod.java:56)
    at tk.wurst_client.events.UpdateEvent.fire(UpdateEvent.java:22)
    at tk.wurst_client.events.EventManager.fire(EventManager.java:45)
    at net.minecraft.client.entity.EntityPlayerSP.onUpdate(EntityPlayerSP.java:132)
    at net.minecraft.world.World.updateEntityWithOptionalForce(World.java:1884)
    at net.minecraft.world.World.updateEntity(World.java:1853)
    at net.minecraft.world.World.updateEntities(World.java:1702)
    at net.minecraft.client.Minecraft.runTick(Minecraft.java:2297)
    at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1192)
    at net.minecraft.client.Minecraft.run(Minecraft.java:372)
    at net.minecraft.client.main.Main.main(Main.java:129)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at net.minecraft.entity.Entity.getDistanceToEntity(Entity.java:1351)
    at tk.wurst_client.mods.KillauraLegitMod.onUpdate(KillauraLegitMod.java:56)
    at tk.wurst_client.events.UpdateEvent.fire(UpdateEvent.java:22)

-- Affected event --
Details:
    Event class: tk.wurst_client.events.UpdateEvent
Stacktrace:
    at tk.wurst_client.events.EventManager.fire(EventManager.java:45)
    at net.minecraft.client.entity.EntityPlayerSP.onUpdate(EntityPlayerSP.java:132)
    at net.minecraft.world.World.updateEntityWithOptionalForce(World.java:1884)
    at net.minecraft.world.World.updateEntity(World.java:1853)

-- Entity being ticked --
Details:
    Entity Type: null (net.minecraft.client.entity.EntityPlayerSP)
    Entity ID: 7615
    Entity Name: obiobiminecraft
    Entity's Exact location: -614,59, 5,00, 631,74
    Entity's Block location: -615,00,5,00,631,00 - World: (-615,5,631), Chunk: (at 9,0,7 in -39,39; contains blocks -624,0,624 to -609,255,639), Region: (-2,1; contains chunks -64,32 to -33,63, blocks -1024,0,512 to -513,255,1023)
    Entity's Momentum: 0,32, 0,16, 0,03
    Entity's Rider: ~~ERROR~~ NullPointerException: null
    Entity's Vehicle: ~~ERROR~~ NullPointerException: null
Stacktrace:
    at net.minecraft.world.World.updateEntities(World.java:1702)

-- Affected level --
Details:
    Level name: MpServer
    All players: 4 total; [EntityPlayerSP['obiobiminecraft'/7615, l='MpServer', x=-614,59, y=5,00, z=631,74], EntityOtherPlayerMP['Csabika1522'/6883345, l='MpServer', x=-655,34, y=6,00, z=659,84], EntityOtherPlayerMP['Basa2005'/6905904, l='MpServer', x=-658,16, y=6,00, z=645,97], EntityOtherPlayerMP['hgamer'/6651499, l='MpServer', x=-659,25, y=4,75, z=657,69]]
    Chunk stats: MultiplayerChunkCache: 437, 437
    Level seed: 0
    Level generator: ID 01 - flat, ver 0. Features enabled: false
    Level generator options: 
    Level spawn location: -665,00,6,00,636,00 - World: (-665,6,636), Chunk: (at 7,0,12 in -42,39; contains blocks -672,0,624 to -657,255,639), Region: (-2,1; contains chunks -64,32 to -33,63, blocks -1024,0,512 to -513,255,1023)
    Level time: 1317795942 game time, 91896000 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
    Forced entities: 50 total; [EntityOtherPlayerMP['KinGeri'/6137172, l='MpServer', x=-639,72, y=4,00, z=678,44], EntityOtherPlayerMP['hgamer'/6651499, l='MpServer', x=-659,25, y=4,75, z=657,69], EntityArrow['arrow'/6924925, l='MpServer', x=-589,19, y=4,04, z=695,54], EntityOtherPlayerMP['hgamer'/6651499, l='MpServer', x=-656,63, y=6,00, z=661,97], EntityArrow['arrow'/6928729, l='MpServer', x=-603,63, y=5,00, z=679,34], EntityOtherPlayerMP['ClauMc'/6670925, l='MpServer', x=-665,08, y=6,00, z=636,56], EntityOtherPlayerMP['zolika200512'/6831033, l='MpServer', x=-665,00, y=6,00, z=636,00], EntityArrow['arrow'/6932564, l='MpServer', x=-600,00, y=4,03, z=693,53], EntityArrow['arrow'/6927416, l='MpServer', x=-593,13, y=4,03, z=689,28], EntityOtherPlayerMP['Basa2005'/6905904, l='MpServer', x=-665,72, y=5,02, z=663,06], EntityOtherPlayerMP['gabajaking'/6536217, l='MpServer', x=-665,00, y=6,02, z=636,00], EntityOtherPlayerMP['Basa2005'/6905904, l='MpServer', x=-658,16, y=6,00, z=645,97], EntityPlayerSP['obiobiminecraft'/7615, l='MpServer', x=-614,59, y=5,00, z=631,74], EntityOtherPlayerMP['maygar2'/6682476, l='MpServer', x=-631,34, y=4,00, z=679,80], EntityOtherPlayerMP['gaborlesnyik'/6584311, l='MpServer', x=-651,72, y=4,48, z=680,66], EntityOtherPlayerMP['MrSweet'/6630824, l='MpServer', x=-605,28, y=4,47, z=667,69], EntityArrow['arrow'/6934557, l='MpServer', x=-598,34, y=4,03, z=694,69], EntityOtherPlayerMP['Novasty'/5372473, l='MpServer', x=-665,03, y=6,00, z=634,69], EntityOtherPlayerMP['HunMax00'/6408051, l='MpServer', x=-684,36, y=4,00, z=679,34], EntityOtherPlayerMP['Csabika1522'/6883345, l='MpServer', x=-655,34, y=6,00, z=659,84], EntityOtherPlayerMP['hgamer'/6651499, l='MpServer', x=-659,25, y=4,75, z=657,69], EntityOtherPlayerMP['KinGeri'/6137172, l='MpServer', x=-642,05, y=4,72, z=683,30], EntityOtherPlayerMP['csabi552'/6830425, l='MpServer', x=-682,97, y=4,02, z=657,69], EntityOtherPlayerMP['Csabika1522'/6883345, l='MpServer', x=-655,34, y=6,00, z=659,84], EntityOtherPlayerMP['alphanova'/6836842, l='MpServer', x=-655,47, y=6,00, z=626,84], EntityOtherPlayerMP['hgamer'/6651499, l='MpServer', x=-655,31, y=6,00, z=661,13], EntityArrow['arrow'/6922466, l='MpServer', x=-658,63, y=4,00, z=671,28], EntityOtherPlayerMP['TriOliver'/5985245, l='MpServer', x=-664,53, y=4,50, z=676,78], EntityArrow['arrow'/6929892, l='MpServer', x=-600,59, y=4,03, z=691,28], EntityOtherPlayerMP['zolika200512'/6831033, l='MpServer', x=-663,31, y=4,00, z=652,03], EntityItem['item.item.helmetIron'/6932977, l='MpServer', x=-647,69, y=4,00, z=657,09], EntityOtherPlayerMP['BB005'/6582074, l='MpServer', x=-702,24, y=4,61, z=684,25], EntityOtherPlayerMP['kelekecske'/4963618, l='MpServer', x=-665,91, y=5,16, z=678,83], EntityOtherPlayerMP['kecskevok68'/5792278, l='MpServer', x=-666,63, y=6,00, z=646,34], EntityOtherPlayerMP['csabi552'/6830425, l='MpServer', x=-660,66, y=4,02, z=661,22], EntityOtherPlayerMP['gabajaking'/6536217, l='MpServer', x=-651,81, y=5,84, z=683,05], EntityArrow['arrow'/6908867, l='MpServer', x=-642,41, y=4,03, z=693,66], EntityOtherPlayerMP['CCSabi112'/6878555, l='MpServer', x=-664,91, y=6,00, z=640,13], EntityOtherPlayerMP['gaborlesnyik'/6584311, l='MpServer', x=-665,00, y=6,00, z=636,00], EntityOtherPlayerMP['CCSabi112'/6878555, l='MpServer', x=-681,33, y=4,16, z=668,35], EntityOtherPlayerMP['4REAL'/6674992, l='MpServer', x=-680,22, y=5,81, z=662,79], EntityOtherPlayerMP['csabi552'/6830425, l='MpServer', x=-682,69, y=5,00, z=677,25], EntityOtherPlayerMP['zolika200512'/6831033, l='MpServer', x=-685,07, y=5,39, z=680,95], EntityArrow['arrow'/6910397, l='MpServer', x=-583,22, y=4,00, z=692,22], EntityArrow['arrow'/6926518, l='MpServer', x=-592,94, y=4,03, z=694,72], EntityItemFrame['entity.ItemFrame.name'/480, l='MpServer', x=-586,50, y=6,50, z=641,03], EntityOtherPlayerMP['Basa2005'/6905904, l='MpServer', x=-651,91, y=4,00, z=657,50], EntityArrow['arrow'/6928006, l='MpServer', x=-593,25, y=4,03, z=690,75], EntityOtherPlayerMP['csabi552'/6830425, l='MpServer', x=-661,38, y=4,02, z=664,03], EntityOtherPlayerMP['Basa2005'/6905904, l='MpServer', x=-658,16, y=6,00, z=645,97]]
    Retry entities: 0 total; []
    Server brand: BungeeCord (git:BungeeCord-Bootstrap:1.8-SNAPSHOT:1c4c1da:unknown) <- Spigot
    Server type: Non-integrated multiplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.addWorldInfoToCrashReport(WorldClient.java:399)
    at net.minecraft.client.Minecraft.addGraphicsAndWorldToCrashReport(Minecraft.java:2834)
    at net.minecraft.client.Minecraft.run(Minecraft.java:394)
    at net.minecraft.client.main.Main.main(Main.java:129)

-- System Details --
Details:
    Minecraft Version: 1.8
    Wurst Client Version: 2.22 (latest: 2.22)
    Operating System: Windows 7 (amd64) version 6.1
    Java Version: 1.8.0_102, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 192344968 bytes (183 MB) / 369627136 bytes (352 MB) up to 1060372480 bytes (1011 MB)
    JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    Launched Version: Wurst
    LWJGL: 2.9.1
    OpenGL: Mobile Intel(R) 4 Series Express Chipset Family GL version 2.1.0 - Build 8.15.10.2869, Intel
    GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because EXT_framebuffer_object is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

Using VBOs: Yes
Is Modded: Very likely; Jar signature invalidated
Type: Client (map_client.txt)
Resource Packs: []
Current Language: English (US)
Profiler Position: N/A (disabled)
```

#### Other Information
- Wurst Client version:  2.22
- Operating system: Windows 7

---

I have read and understood the [Bug Tracker Rules](https://github.com/Wurst-Imperium/Wurst-Client-Bugs/blob/master/.github/CONTRIBUTING.md) before posting this bug report and filled out the template above with the correct information.

**Comment by galaxypigs** [comment]
same please help i cant use worst anymore

**Comment by Edtlogin** [comment]
I got the same problem.

**Comment by RobbieDragneel** [comment]
When will someone post how to fix this?

**Comment by RobbieDragneel** [comment]
To fix this, Press the Windows key and R to open the Run box.

Type %appdata%\.minecraft\wurst\modules.json

Open it with notepad, and look for KillAuraLegit
(If it says true, change it to false and save)

**Comment by Edtlogin** [comment]
If it didnt say true...

**Comment by Alexander01998** [comment]
Should be fixed in v2.23.
